Libraries/Opc.Ua.Server/Session/Session.cs

```diff
@@ -566,6 +660,16 @@ public bool UpdateLocaleIds(StringCollection localeIds)
         /// <summary>
         /// Activates the session and binds it to the current secure channel.
         /// </summary>
+#if ECC_SUPPORT
+        public bool Activate(
+            OperationContext context,
+            List<SoftwareCertificate> clientSoftwareCertificates,
+            UserIdentityToken identityToken,
+            IUserIdentity identity,
+            IUserIdentity effectiveIdentity,
+            StringCollection localeIds,
+            Nonce serverNonce)
```

Comment:
could the new Nonce class also be used for legacy code? Then there would not be different functions.